src/main/java/uk/gov/hmcts/divorce/systemupdate/service/CcdSearchService.java

```diff
@@ -319,4 +319,25 @@ public List<CaseDetails> searchForCases(
             sourceBuilder.toString()
         ).getCases();
     }
+
+    public List<CaseDetails> searchJointApplicationsWithAccessCodePostIssueApplication(User user, String serviceAuth) {
+
+        final SearchSourceBuilder sourceBuilder = SearchSourceBuilder
+            .searchSource()
+            .query(
+                boolQuery()
+                    .must(boolQuery().must(existsQuery("data.accessCode")))
+                    .must(boolQuery().must(existsQuery("data.issueDate")))
+                    .must(boolQuery().must(termsQuery("data.applicationType", "jointApplication")))
+                    )
```

Comment:
I think it can be simplified even further to:
`boolQuery()
        .must(existsQuery("data.accessCode"))
        .must(existsQuery("data.issueDate"))
        .must(termsQuery("data.applicationType", "jointApplication"))`

Reply:
done